.github/workflows/cleanup.yaml

```diff
@@ -0,0 +1,30 @@
+name: GCR Cleanup
+on:
+  schedule:
+    - cron: '0 0 * * *' # Every day at midnight
+  workflow_dispatch:
```

Comment:
Not sure if this works yet because you can only run it manually from the default branch, I figured we could merge this in then I can create a follow up story to get it fully working.

Reply:
Thank you so much to getting to this @pdeziel &mdash; this is some of our oldest devOps tech debt!